app/controllers/pools_controller.rb

```diff
@@ -0,0 +1,26 @@
+class PoolsController < ApplicationController
+  def create
+    name = "Pool #{Tournament.find(pool_params[:tournament_id]).pools.count + 1}"
+    @pool = Pool.create(name: name, tournament_id: pool_params[:tournament_id])
+    if @pool.save
+      redirect_to tournament_pools_path(@pool.tournament_id)
+    else
+      flash.now[:errors] = @pool.errors.full_messages
+      render :new
+    end
+  end
+
+  def destroy
+    @pool = Pool.find(params[:id])
+    if @pool.name == 'Unassigned'
```

Comment:
You can put this as `DEFAULT_POOL = 'Unassigned'` in the Pool model, and then call it `Pool::DEFAULT_POOL`. (Alternatively, you can put it in a helper method... we should discuss this strategy.)